examples/forms.amp.html

```diff
@@ -57,6 +57,34 @@
             color: #dc4e41;
         }
 
+        .verification-form label {
```

Comment:
less so for this example page and more so to kick of the design for AmpByExample and/or AMP Start.